webui/react/src/services/api.ts

```diff
@@ -54,61 +55,33 @@ export const getAgents = generateApi<EmptyParams, Agent[]>(Config.getAgents);
 
 /* Experiments */
 
-type ExperimentListSortKey =
-  | 'SORT_BY_UNSPECIFIED'
-  | 'SORT_BY_ID'
-  | 'SORT_BY_DESCRIPTION'
-  | 'SORT_BY_START_TIME'
-  | 'SORT_BY_END_TIME'
-  | 'SORT_BY_STATE'
-  | 'SORT_BY_NUM_TRIALS'
-  | 'SORT_BY_PROGRESS'
-  | 'SORT_BY_USER';
-
-const experimentSortKeys = {
-  endTime: 'SORT_BY_END_TIME',
-  id: 'SORT_BY_ID',
-  name: 'SORT_BY_DESCRIPTION',
-  numTrials: 'SORT_BY_NUM_TRIALS',
-  progress: 'SORT_BY_PROGRESS',
-  startTime: 'SORT_BY_START_TIME',
-  state: 'SORT_BY_STATE',
-  user: 'SORT_BY_USER',
-};
-type ExperimentSortKey = keyof typeof experimentSortKeys;
-
-type ExperimentListState =
-  | 'STATE_UNSPECIFIED'
-  | 'STATE_ACTIVE'
-  | 'STATE_PAUSED'
-  | 'STATE_STOPPING_COMPLETED'
-  | 'STATE_STOPPING_CANCELED'
-  | 'STATE_STOPPING_ERROR'
-  | 'STATE_COMPLETED'
-  | 'STATE_CANCELED'
-  | 'STATE_ERROR'
-  | 'STATE_DELETED';
-
 export const getExperimentList = async (
-  sorter: TableSorter,
-  pagination: TablePagination,
+  sorter: ApiSorter<V1GetExperimentsRequestSortBy>,
+  pagination: Pagination,
   filters: ExperimentFilters,
   search?: string,
-): Promise<V1GetExperimentsResponse> => {
+): Promise<{ experiments: ExperimentItem[], pagination?: V1Pagination }> => {
   try {
+    const sortBy = Object.values(V1GetExperimentsRequestSortBy).includes(sorter.key) ?
+      sorter.key : V1GetExperimentsRequestSortBy.UNSPECIFIED;
+
+    /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
non-blocking: considering we're gonna be defining this type of any in many of our other API calls

can we disable any check only line by line or use a type alias 
`type SwaggerEnum = any` or sth like that to be a bit safer?